File: kildey004/programacaobasica/questao9.py
print("**calculadora de gastos**")

x = float(input("digite quanto voce tem para gastar em R$: "))

xs = float(input("digite quanto era um salgado: "))

sa = float(input("digite quanto e um salgado agora: "))

tsa = x / sa

txs = x / xs

print( "antes voce compraria " + str(txs) + " salgados")
print("agora voce pode comprar " + str(tsa) + " salgados")

File: kildey004/programacaobasica/Questao9.java
import java.util.Scanner;

public class Questao9 {
    public static void main (String[] args) {
        double x;
        double xs;
        double sa;
        double tsa;
        double txs;
        Scanner s;

        System.out.println("**CALCULADORA DE GASTOS**");

        s =  new Scanner(System.in);
        System.out.println("Digite o quanto voce tem pra gastar em R$:");
        x = s.nextDouble();

        System.out.println("Digite quanto era um salgado:");
        xs = s.nextDouble();

        System.out.println("Digite quanto eh um salgado agora:");
        sa = s.nextDouble();

        tsa = x / sa;
        txs = x / xs;

        System.out.println("Antes voce compraria " + txs + " salgados");
        System.out.println("Agora voce pode comprar " + tsa + " salgados");

    }
}
